### ISSUE TITLE
User auth field not documented in OPTIONS

### ISSUE THREAD

**Issue by AlanCoding** [ansible/awx#2301]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
 - API

##### SUMMARY
The user serializer provides the `auth` field but it is not present in OPTIONS

##### ENVIRONMENT
* AWX version: 1.0.8
* AWX install method: openshift, minishift, docker on linux, docker for mac, boot2docker
* Ansible version:  X.Y.Z
* Operating System:
* Web Browser:

##### STEPS TO REPRODUCE

GET `/api/v2/users/`

OPTIONS `/api/v2/users/`

##### EXPECTED RESULTS

Fields match, GET vs. OPTIONS

##### ACTUAL RESULTS

`auth` field returned but not in options.

Also, the field is not present in the access_list endpoints, and I would argue that it should be.

**Comment by rooftopcellist** [comment]
@thedoubl3j here is where the auth field is added to the user serializers: https://github.com/ansible/awx/blob/devel/awx/api/serializers.py#L884

It looks like this field should contain the SSO providers that user was authenticated with.

**Comment by harshakhmk** [comment]
Does the issue still needs to be addressed, if yes I can contribute

**Comment by thedoubl3j** [comment]
@harshakhmk yes, I didn't realize I was still assigned. Since the issue is still open, the issue still remains. feel free to open a PR and reference this issue. how we might want to implement might have changed but even then, I doubt it would be a large shift. 

edit: not a bug but rather an enhancement so wording changed
